.vscode/settings.json

```diff
@@ -22,7 +22,7 @@
   },
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "eslint.format.enable": true,
-  "breadcrumbs.enabled": false,
+  "breadcrumbs.enabled": true,
```

Comment:
Is this required?

Reply:
yes please, i would like for this to be merged over too, it's very annoying not to have the breadcrumbs when u have opened files in vs code